site/content/docs/5.2/utilities/colors.md

```diff
@@ -23,6 +23,9 @@ Colorize text with color utilities. If you want to colorize links, you can use t
 <p class="text-light">.text-light</p>
```

Comment:
`.text-secondary` isn't defined in Boosted, is there anything to do in here ? (remove the doc or add the utility)

Reply:
Let's create an issue to tackle after this PR someday. The issue is already there in Boosted for a long time.